DH_Engine/Classes/DHVoiceChatRoom.uc

```diff
@@ -54,6 +95,29 @@ simulated event bool IsMember(PlayerReplicationInfo PRI, optional bool bNoCascad
     return bool(GetMask() & (1 << PRI.VoiceID));
 }
 
+simulated function array<PlayerReplicationInfo> GetMembers()
+{
+    local array<PlayerReplicationInfo>      PRIArray;
+    local int                               i;
+
+    if (GRI != none && ValidMask())
+    {
+        for (i = 0; i < GRI.PRIArray.length; ++i)
+        {
+            if (IsMember(GRI.PRIArray[i]))
```

Comment:
Both this `if` statement and the `else if` statement ends up doing the exact same thing. Combine them into one conditional statement.